build_test/lib/src/package_reader.dart

```diff
@@ -74,26 +74,21 @@ class PackageAssetReader extends AssetReader
   }
 
   File _resolve(AssetId id) {
-    final packagePath = _packageResolver.packagePath(id.package);
-    if (packagePath == null) {
-      return null;
+    final uri = id.uri;
+    if (uri.isScheme('package')) {
+      return new File(_packageResolver.resolveUri(id.uri).path);
```

Comment:
Using the `path` from an uri fails under windows.

Doing something like 

```dart
      final file = new File(new File('.').absolute.uri.path);
      file.readAsString();
```

produces a `(OS Error: The filename, directory name, or volume label syntax is incorrect.
, errno = 123)` because the path start with a leading `/`.

Reply:
Can you use 
```dart
return new File.fromUri(_packageResolver.resolveUri(id.uri));
``` 
instead?

Reply:
Thanks!

Can you try with https://github.com/dart-lang/build/pull/1294 and see if it's working for you?

Reply:
Tested and with this change it's working again.